tests/integration/test_sqs.py

```diff
@@ -25,3 +26,31 @@ def test_create_fifo_queue(self):
 
         # it should preserve .fifo in the queue name
         assert fifo_queue in queue_url
+
+    def test_sqs_listener_message_attrs(self):
```

Comment:
Given that this is more of a unit test, we could also put this into a new file `tests/unit/test_sqs.py`

Reply:
Sure thing!